sui/utils/utils.js

```diff
@@ -270,6 +270,17 @@ const checkTrustedAddresses = (trustedAddresses, destinationChain) => {
     }
 };
 
+const saveGeneratedTx = async (tx, message, client, options) => {
+    const { txFilePath } = options;
+    validateParameters({ isNonEmptyString: { txFilePath } });
+
+    const txBytes = await tx.build({ client });
+    const txB64Bytes = toB64(txBytes);
+
+    writeJSON({ message, status: 'PENDING', unsignedTx: txB64Bytes }, txFilePath);
```

Comment:
do we need `status: PENDING` here or we could simply delete the whole file after broadcasting?